lucida/commandcenter/controllers/Config.py

```diff
@@ -12,41 +12,163 @@
 # models are already saved in `../models`.
 TRAIN_OR_LOAD = 'train' # either 'train' or 'load'
 
+
+
+
+
+#Contains serviceName and data to pass. Needed for batch (and thereby parallel) processing.
+class serviceRequestData(object):
+
+	def __init__(self,nameOfService,argData):
+		self.argumentData = argData
+		self.serviceName = nameOfService
+	
+
+class workFlow(object):
+		def __init__(self):
+			self.currentState = 0; # What state on the state graph
+			self.isEnd = False;
+			self.batchedData = []
+	
```

Comment:
just curious why there's so much space here? are you following a specific style? We should start linting.